tests/phpunit/config.php

```diff
@@ -171,7 +171,7 @@ public function setUp(): void {
      * @runInSeparateProcess
      */
     public function test_get_modules() {
-        $config = new Hm_Site_Config_File('./data/siteconfig.rc');
```

Comment:
I think you should also remove that file if it is no longer used.

Reply:
Done.